tests/data/test_data.py

```diff
@@ -31,14 +31,18 @@ def test_is_uri():
     assert is_uri("gs://some/gs/path")
     assert is_uri("dbfs:/some/dbfs/path")
     assert is_uri("file://some/local/path")
+    assert is_uri("hdfs://some/hdfs/path")
+    assert is_uri("viewfs://some/viewfs/path")
     assert not is_uri("/tmp/some/local/path")
 
 
```

Comment:
Reminder: Add unit tests for error cases

Reply:
I'm not sure what test cases you want me to add here, could you give me more indications please ?